layouts/index.html

```diff
@@ -1,7 +1,7 @@
 {{ partial "header.html" . }}
 
 <header class="frontpage">
-  <h1>&#x1f680; Camel 3.1 is here! <wbr><a href="blog/Camel3-Whatsnew/">What's new?</a></h1>
+  <h1>&#x1f680; Camel 3.1 is here! <br><a href="blog/Camel3-Whatsnew/">What's new?</a></h1>
```

Comment:
This is intentional, please revert this change.

Reply:
okay